test/reload_system.bats

```diff
@@ -21,12 +24,17 @@ function expect_log_success() {
 	wait_for_log "applied new registry configuration"
 }
 
+function expect_log_warning() {
+	wait_for_log "rate limit exceeded, skipping reload"
+}
+
```

Comment:
I don't see this being used anywhere in this commit.

Reply:
Removed it